### DISCUSSION TITLE
Station not accessible

### DISCUSSION THREAD

**Discussion by ve3hls** [magicbug/Cloudlog#1917] [General]
Hi Group,

I just uploaded an DIF file and was presented with a window with dozens (hundreds?) of "Station not accessible" messages.  This happened the last time I was logged in and Peter walked me through the steps to fix it.  I only wish I could remember what that was!  I checked in a few places about station logs and station locations and everything looks OK, but I can't be sure.

I went to my logbook anyway and also noticed there are no QSOs listed after May 2022, which is alarming.

Any assistance would be gratefully received.

Ken Alexander

**Comment by ve3hls** [comment]
Nobody?  Nobody knows anything about this error message?  Cloudlog is unusable unless I can fix this and, as I mentioned before, everything looks OK to me.

Ken

**Reply by phl0** [reply]
It seems that you tried to upload the ADIF file to a station location that does not belong to the currently logged in user. I think you might have to assign the location to the user.

**Comment by ve3hls** [comment]
Hi Florian,

I had the exact same problem a few weeks ago.  Peter walked me through th
setup and everything worked Ok after that.  That said and done I don't
understand why it's happening again.  I checked the station setups and they
appear to be unchanged.

Ken  VE3HLS

**Reply by magicbug** [reply]
Can walk you through it again tomorrow if that works Ken? not having a good health day today drop me a note on Twitter to my main account not the Cloudlog one.

**Reply by ve3hls** [reply]
Peter, I'm sorry you're not feeling well.  I sent you a Direct Message via Twitter at least once already.  I deleted my station location and rebuilding it.  That did nothing.  Haven't been able to use Cloudlog for a few months now, but the QSOs keep piling up.

Do you have any knowledgeable colleagues?  Someone who can do more than tell me what I already know?

Thanks,

Ken

**Reply by phl0** [reply]
Hej Ken, I am afraid it would require access to the database to see what's wrong. Are you hosting it yourself or do you use cloudlog.co.uk's service by @magicbug?

**Comment by ve3hls** [comment]
Hi Peter,

I'd appreciate that.  I did go back and double check the setups and I
didn't notice anything was missing or incorrect.

Let me know when you're feeling up to it.

Thanks and 73,

Ken  VE3HLS

**Comment by ve3hls** [comment]
Hi Florian,

Peter is hosting it for me.  Any help would be appreciated.

73,
Ken  VE3HLS

**Reply by phl0** [reply]
Sorry to say but I am afraid only he can have a look at the database then if something is wrong ... :(

**Comment by ve3hls** [comment]
Are you able to check if my Station and (what is it?) Location are
configured correctly Flo?  Peter and I re-did that last time and everything
worked OK (for a little while anyway) after that.

Thanks,
Ken  VE3HLS

**Reply by phl0** [reply]
I think I could.

**Comment by ve3hls** [comment]
Ok, great!  You can contact Peter or me if you need my password to get in.

Ken  VE3HLS

**Comment by magicbug** [comment]
I cant see any issue logbook is displayed like I'd expect it

![image](https://user-images.githubusercontent.com/84308/224324395-b3f4e6f1-00df-4830-a262-e1059bcc1692.png)

ADIFs import correctly when I select the station location for the QSOs and then provide the file.

**Comment by magicbug** [comment]
If your trying to use the API feature for WSJT-X QSOs then you need to make sure your using the up to date API key displayed under https://ve3hls.cloudlog.co.uk/index.php/api/help

**Comment by ve3hls** [comment]
Hi Peter,

I do't know what to make of that.  I have imported ADIF files from WSJT-X
and N1MM contest logger and I get the same error message for each log it
imports.  Also, when I view the logs nothing newer than late May 2022
appears anywhere.  It looks like nothing has been imported since then.

Ken  VE3HLS

**Reply by magicbug** [reply]
Can you email me the ADIFs so I can look at them

**Comment by ve3hls** [comment]
Sorry.  I don't know what an API feature is.  I import the ADIF file WSJT-X
creates.

Ken  VE3HLS

**Reply by magicbug** [reply]
In that case API stuff doesn't matter

**Comment by ve3hls** [comment]
Hi Peter,

Just finished breakfast and made my way out to the shack.  I'll see what I
can dig out (and try it myself first) and get back to you shortly.  Looks
like it's about 3:00 AM out your way, so I have time!

73,
Ken  VE3HLS

**Comment by ve3hls** [comment]
OK, this is strange.  I created a new .ADI file in WSJT-X and imported it
into Cloudlog.  This
has several ATNOs in it (I had a good day a few days ago).

After importing, Cloudlog reports that everything I uploaded was a
duplicate log.
There's still nothing newer than May 28, 2022 in Cloudlog.
I reviewed the WSJT-X .ADI file I imported and it contains all logs up to
March 7, 2023, my most recent QSO.

That's 10 month's worth of QSOs missing.

I don't see that missing station error message after importing.  That's a
stop in the right direction!

73,
Ken  VE3HLS

**Comment by ve3hls** [comment]
OK.  This is also interesting!  The recently uploaded .ADI logs don't
appear in my Cloudlog log, but they DO appear when I create a DXCC Timeline
in Cloudlog.  So they are in there somewhere but not in the main log.

73,
Ken  VE3HLS

**Comment by ve3hls** [comment]
Stranger and stranger!  Now EVERYTHING is in my logbook!  It's completely
up to date!  (are you guys quietly doing things in the background that I
can't see?)

73,
Ken  VE3HLS
